.github/workflows/tests.yml

```diff
@@ -46,6 +46,10 @@ jobs:
           # Test with minimum dependencies versions
           - {impl: c, python: "3.8", ext: min, postgres: "postgres:15"}
 
+          # Test with PyPy.
+          - {impl: python, python: "pypy3.9", postgres: "postgres:13"}
+          - {impl: python, python: "pypy3.10", postgres: "postgres:14"}
```

Comment:
I didn't try locally with 3.9, but hoping it should work.

Have used the same versions of PostgreSQL as for the CPython ones above.